src/main/kotlin/org/jmailen/gradle/kotlinter/KotlinterPlugin.kt

```diff
@@ -42,8 +42,8 @@ class KotlinterPlugin : Plugin<Project> {
                         lintTask.ignoreFailures.set(provider { kotlinterExtension.ignoreFailures })
                         lintTask.reports.set(
                             provider {
-                                kotlinterExtension.reporters.associate { reporter ->
-                                    reporter to reportFile("$id-lint.${reporterFileExtension(reporter)}")
+                                kotlinterExtension.reporters.associateWith { reporter ->
```

Comment:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/36954793/175789640-cdc20855-b5c7-43f4-aeb7-f4196f9e1051.png">

I navigate through IDE via `F2` (next highlighted error) so I usually try to fix things like this 😄